_examples/idv/handlers.session.go

```diff
@@ -208,3 +212,83 @@ func showPrivacyPolicyPage(c *gin.Context) {
 	render(c, gin.H{}, "privacy.html")
 	return
 }
+
+/* For Testing */
+
+func showDigitalPage(c *gin.Context) {
+	sessionReq, err := buildDigitalIdentitySessionReq()
+	if err != nil {
+		c.HTML(
+			http.StatusInternalServerError,
+			"error.html",
+			gin.H{
+				"ErrorTitle":   "Error when building sessions spec",
+				"ErrorMessage": err.Error()})
+		return
+	}
+	pageFromShareSessionReq(c, sessionReq)
+}
+
+func initialiseDigitalIdentityClient() error {
+	var err error
+	sdkID = os.Getenv("YOTI_CLIENT_SDK_ID")
+	keyFilePath := os.Getenv("YOTI_KEY_FILE_PATH")
+	key, err = os.ReadFile(keyFilePath)
+	if err != nil {
+		return fmt.Errorf("failed to get key from YOTI_KEY_FILE_PATH :: %w", err)
+	}
+
+	didClient, err = yoti.NewDigitalIdentityClient(sdkID, key)
+	if err != nil {
+		return fmt.Errorf("failed to initialise Share client :: %w", err)
+	}
+	//didClient.OverrideAPIURL("https://connect.public.stg1.dmz.yoti.com/share/")
```

Comment:
for testing with different envs like stg, might be better to put `YOTI_API_URL` in the `.env` file so it will just pick from there and you won't have to change the code you commit

Reply:
local temporary tests and removed